model/docs/pyrenew_demo.qmd

```diff
@@ -166,6 +181,8 @@ hospmodel.run(
     rng_key=jax.random.PRNGKey(54),
     mcmc_args=dict(progress_bar=False),
     )
+
+# Damon: What is the relationship between `n_timepoints` and `observed_hospitalizations`? Is `n_timepoints` always one less than the number of hospitalization observtion times? If so, is this parameter redundant?
```

Comment:
Great points, that's something that needs to be addressed. When observed hospitalizations are passed, `n_timepoints` should be directly computed.